src/components/Modal/Modal/index.tsx

```diff
@@ -0,0 +1,38 @@
+import Portal from '../Potal';
+import styles from './Modal.module.scss';
+import useCloseModal from '@/hooks/useModalClose';
+import { useEffect, useRef } from 'react';
+
+type ModalProps = {
+  showModal: boolean;
+  handleClose: () => void;
+  children: React.ReactNode;
+};
+
+function Modal({ showModal, handleClose, children }: ModalProps) {
+  const modalRef: any = useRef();
+  useCloseModal(showModal, handleClose, modalRef);
+
+  useEffect(() => {
+    if (showModal) {
+      document.body.style.cssText = `overflow: hidden;`;
+      return () => {
+        document.body.style.cssText = `overflow: scroll;`;
```

Comment:
```suggestion
      document.body.style.cssText = `overflow: hidden;`;
      return () => {
        document.body.style.cssText = `overflow: scroll;`;
```
사소하지만 여기 문자열로 바꿔도 괜찮을 것 같네요! `'overflow: scroll'`